ci/jenkins/test-vmc.sh

```diff
@@ -298,7 +298,11 @@ function deliver_antrea {
             VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" make && break
         fi
     done
-    VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" make flow-aggregator-ubuntu
+    if [[ "$COVERAGE" == true ]]; then
+      VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" make flow-aggregator-ubuntu
```

Comment:
swap L302 and L304?

Reply:
Yes, thats right. It was a mistake. Done.